includes/footer.php

```diff
@@ -29,6 +29,10 @@
         <small class="d-flex mb-3">Supported by<div class="social-icons">
             <a href="/about#czi-eoss">
               <img src="/assets/img/contributors-colour/CZI-alt.svg" alt="CZI" class="" style="max-width: 50px">
+            </a>,
+            <a href="/about#microsoft-azure">
+              <img src="/assets/img/contributors-white/azure.svg" alt="Microsoft Azure" class="hide-light" style="max-width: 50px">
+              <img src="/assets/img/contributors-colour/azure.svg" alt="Microsoft Azure" class="hide-dark" style="max-width: 50px">
```

Comment:
Because it is the same logo in both, we don't need two declarations
```suggestion
              <img src="/assets/img/contributors-colour/azure.svg" alt="Microsoft Azure" style="max-width: 50px">
```